cli.sh

```diff
@@ -49,12 +49,6 @@ elif [ "$ACTIONS" == "$DEPLOY" ]; then
     ACTIONS=$(gum choose --cursor-prefix "[ ] " --selected-prefix "[✓] " --no-limit "$LOGIN_IN_VM" "$DEPLOY_SPA")
 
     if [ "$ACTIONS" == "$LOGIN_IN_VM" ]; then
-    echo "What's the user of the $(gum style --foreground 212 "Virtual Machine") (in most cases it's $(gum style --foreground 212 "root")):"
-    read USER 
-
-    echo "What's the ip of the $(gum style --foreground 212 "Virtual Machine")":
-    read IP 
-
-    ssh "$USER@$IP";
+        ssh $HUB_VM
```

Comment:
The idea for this is to check if $HUB_VM has a value and do `ssh $HUB_VM` when there is one, otherwise old steps are to be executed.

You need a second option which is going to replace DEPLOY_SPA which will essentially going to do the same with the difference that it's going to run the `set_vm_env` script when it successfully connects to the VM

Reply:
Should I bother doing the second option now or do I leave it for later? I think it'd be best for me to do it prior to us actually deploying the website tomorrow